system-test/storage.ts

```diff
@@ -3923,7 +3925,8 @@ describe('storage', () => {
       } else {
         return false;
       }
-    } catch (error) {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    } catch (error: any) {
```

Comment:
should this be `any` or `Error`?

Reply:
So you can't actually declare it as an `Error` in the catch block, typescript will only take `unknown` or `any`. I can try one other thing to. try and figure out what type this is so it can be properly casted below.